Data Generator.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+first_names = ["James", "Mary", "Liam", "Olivia", "Robert", "Patricia", "Noah", "Emma", "John", "Jennifer"]
+middle_names = ["Louise", "Rose", "Grace", "William", "Jane", "Elizabeth", "Thomas", "Anne", "Ann", "David"]
+last_names = ["Smith", "Johnson", "Williams", "Brown", "Jones", "Garcia", "Miller", "Davis", "Rodriguez", "Martinez"]
+
+names = []
+
+for i in range(10):
+    for j in range(10):

```

Comment:
These shouldn't be using hardcoded length values. Ideally, we would be iterating over the arrays directly, rather than by index:
```suggestion
for first_name in first_names:
    for last_name in last_names:
```